src/basic/selinux-util.c

```diff
@@ -225,7 +217,7 @@ int mac_selinux_get_create_label_from_exe(const char *exe, char **label) {
                 return -errno;
 
         sclass = string_to_security_class("process");
-        r = security_compute_create(mycon, fcon, sclass, (security_context_t *) label);
+        r = security_compute_create_raw(mycon, fcon, sclass, (security_context_t *) label);
```

Comment:
Hm, it's strange.
For example: `systemd` [calls](https://github.com/systemd/systemd/blob/c3dacc8bbf2dc2f5d498072418289c3ba79160ac/src/core/main.c#L1359) `mac_selinux_setup`. `mac_selinux_setup` [calls](https://github.com/systemd/systemd/blob/c3dacc8bbf2dc2f5d498072418289c3ba79160ac/src/core/selinux-setup.c#L91) `setcon`. `setcon` calls `selinux_raw_to_trans_context`.